### PR TITLE
Feature/th fetch class

### LINKED ISSUE #11: Fetch data

### PR DESCRIPTION
**Pull request by relyt4me**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling- no new features

### Detailed Description / What does this PR do?
Creates the Fetch Class that will be used for all fetching needs. Specifically creates a method for getting an array of all movies. Created a test file for this that we may need to use later after researching the process alongside react.

### Files modified:
Fetch.js and Fetch.test.js

### Has This Been Tested?
- [x] No
- [ ] Yes

### Are there specific methods for testing?
Not Yet

### What are the relevant tickets?
resolves #11 

### Checklist:
- [ ] My code has no unused/commented out code
- [ ] My code has no console logs
- [ ] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/Fetch.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class Fetch {
+  constructor() {
+    this.allMovies = [];
```
Comment (NicoleLatifi) [inline src/Fetch.js:3]: I like having this property to make it easier to test if the fetch is working. Will we keep this property once we are using the data in other components?

File: src/Fetch.test.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    //   },
+    //   {
+    //     id: 2,
+    //     poster_path: 'https://image.tmdb.org/t/p/original//eDnHgozW8vfOaLHzfpHluf1GZCW.jpg',
+    //     backdrop_path: 'https://image.tmdb.org/t/p/original//u9YEh2xVAPVTKoaMNlB5tH6pXkm.jpg',
+    //     title: 'Archive',
+    //     average_rating: 4,
+    //     release_date: '2020-08-13',
+    //   },
+    // ];
```
Comment (NicoleLatifi) [inline src/Fetch.test.js:27]: Will we need to have a sample dataset here for the tests we're going to implement?

File: src/Fetch.js (hunk 1 above)
Comment (relyt4me) [reply src/Fetch.js:3]: I think depending on how we use Fetch later we can keep or remove the properties.

**Comment by relyt4me** [comment]
Looks ready for merge @NicoleLatifi

**Pull request merged**